lxd/instances_post.go

```diff
@@ -490,6 +491,17 @@ func createFromCopy(d *Daemon, r *http.Request, projectName string, profiles []a
 		}
 	}
 
+	// Generate a new UUID for the instance when copying the snapshot to a new instance.
+	// We will later use this new instance UUID as a VM Generation ID
+	// in order to differentiate the two instances (before snapshot and after copy to new instance)
```

Comment:
@gabrielmougard is this part actually needed? Did you see the volatile ID stay the same after a copy? (I think there may already be logic to clear most of the volatile keys on copy). Did you confirm this?

Reply:
I didn't check before making that feature.. I'll try now

Reply:
@tomponline so, after doing:

```
$ lxc config show b1 --expanded | cat 'volatile.uuid'
volatile.uuid: 0ea35b34-4ed6-4ec8-aded-bf092e16dc08

$ lxc snapshot b1 snap0

$ lxc restore b1 snap0

$ lxc config show b1 --expanded | cat 'volatile.uuid'
volatile.uuid: 0ea35b34-4ed6-4ec8-aded-bf092e16dc08
```

So, I'd say that it was not updating the UUID

Reply:
you're right.

Reply:
I'm deleting this